packages/website/build/getBasePath.ts

```diff
@@ -3,7 +3,7 @@ const branchName = process.env.BRANCH_NAME || process.env.GITHUB_HEAD_REF || pro
 
 let basePath = '/';
 if (isCI && branchName !== 'master') {
-    basePath = `/feature/${branchName}/`;
+    basePath = `/feature/${branchName.replace('/', '-')}/`;
```

Comment:
You probably want to replace all occurence and not just the first one:
```suggestion
    basePath = `/feature/${branchName.replaceAll('/', '-')}/`;
```

Reply:
Yes indeed thx